addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4901,28 +4903,73 @@ msgctxt "#10008"
 msgid "Play next"
 msgstr ""
 
-#empty strings from id 10009 to 10010
+#. Header label for add-on configuration removal dialog
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10009"
+msgid "Confirm add-on configuration removal"
+msgstr ""
+
+#. Header label for add-on configuration removal selection dialog
+#: xbmc/addons/gui/GUIDialogAddonSettings.cpp
+msgctxt "#10010"
+msgid "Select add-on configuration to remove"
```

Comment:
@AlwinEsch I k ow this PR already went in, but there's a missing `mscstr ""` at line 4916.

Reply:
I will open a PR with a fix.